### ISSUE TITLE
Significance contours shifted when color bar is present

### ISSUE THREAD

**Issue by tgouhier** [tgouhier/biwavelet#42]
The significance contours are shifted around when the color bar is present. Is this due to an upstream change in the fields package and its image.plot function?

**Comment by vsimko** [comment]
reproducible example?

**Comment by tgouhier** [comment]
Here's a reproducible example: 

``` r
# Real data
data(enviro.data)

# Cross-wavelet of MEI and NPGO
xwt.mei.npgo <- xwt(subset(enviro.data, select = c("date", "mei")),
                    subset(enviro.data, select = c("date", "npgo")))

# Make room to the right for the color bar
par(mfrow = c(2, 1), oma = c(0, 0, 0, 1), mar = c(5, 4, 4, 5) + 0.1)
# significance contours, phase arrows and COI shifted with color bar
plot(xwt.mei.npgo, plot.cb = TRUE, plot.phase = TRUE) 
# significance contours, phase arrows and COI in the right location without color bar
plot(xwt.mei.npgo, plot.cb = FALSE, plot.phase = TRUE) 
```

![rplot](https://cloud.githubusercontent.com/assets/2946117/15098517/73f53106-150d-11e6-99ec-b21a0e20e25b.png)

**Comment by tgouhier** [comment]
The issue arose because of upstream changes in the fields package. The fix was easy enough: plot the color bar at the very end once the COI, phase arrows and contours have already been added. This fixes both the displacement error (#42) and the zooming error (#5).
